docs/src/templates/drawerDataGridSample.jsx

```diff
@@ -0,0 +1,166 @@
+import {
+    Button,
+    Drawer,
+    Icon,
+    Input,
+} from 'react-cm-ui';
+import moment from 'moment-timezone';
+import React, { useState } from 'react';
+
+function ModulesDrawerDataGrid() {
+    const [isDrawerOpen, setIsDrawerOpen] = useState(false);
+    const [searchValue] = useState('');
+
+    function onDrawerToggle() {
+        setIsDrawerOpen(!isDrawerOpen);
+    }
+
+    function onClickTest() {
+        console.log('You just clicked the fourth column!'); // eslint-disable-line no-console
+    }
+
+    return (
+        <div>
+            <Button onClick={() => onDrawerToggle()}>Open Drawer</Button>
+
+            <Drawer
+                isOpen={isDrawerOpen}
+                onClose={() => onDrawerToggle()}
+            >
+                <Drawer.TitleBar
+                    closeButton={(
+                        <Icon compact onClick={() => onDrawerToggle()} type="times" />
+                    )}
+                    title="Data Grid"
+                />
+
+                <Drawer.Navigation
+                    columns={[
+                        {
+                            label: 'Button 1',
+                        }, {
+                            label: 'Button 2',
+                        }, {
+                            label: 'Button 3',
+                        }, {
+                            label: 'Button 4',
+                            onClick: () => onClickTest(),
+                        },
+                    ]}
+                />
+
+                <Drawer.ActionBar
+                    columns={[
+                        {
+                            list: [
+                                {
+                                    jsx: (
+                                        <Icon
+                                            onClick={() => {}}
```

Comment:
NAB.  We should probably define a `const noop = () => {};` and use that to avoid having to do this in `render()`.